test/battle/move_effect/fillet_away.c

```diff
@@ -0,0 +1,20 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_FILLET_AWAY].effect == EFFECT_FILLET_AWAY);
+}
+
+SINGLE_BATTLE_TEST("Fillet Away cuts the user's HP in half")
```

Comment:
This should test that the stats have been boosted.

Can you test that this move fails if the HP is too low, or all the stats are at +6 already, and that it succeeds if only some but not all are at +6.